client/src/builder/components/lexical-node-wiki-article-editor.tsx

```diff
@@ -207,33 +206,37 @@ const ArticleInfo = ({
   return (
     <PopoverContent>
       <div className="flex items-center justify-between">
-        <span className="text-muted-foreground">Details</span>
+        {!article ? (
+          <span className="text-muted-foreground">Error</span>
+        ) : (
+          <span className="text-muted-foreground">Details</span>
+        )}
         <div className="flex">
-          <Dialog>
-            <DialogTrigger asChild>
-              <Button size="xs" variant="ghost" disabled={!article}>
-                <EditIcon />
-              </Button>
-            </DialogTrigger>
-            <DialogContent>
-              <DialogTitle>Edit Article Link</DialogTitle>
-              <div className="mt-2">
-                <EditWikiNodeForm
-                  wikiKey={article?.wikiKey}
-                  node={node}
-                  articleKey={articleKey}
-                />
-              </div>
-            </DialogContent>
-          </Dialog>
+          {isLoading || !article || !articleKey ? (
+            <></>
+          ) : (
+            <Dialog>
+              <DialogTrigger asChild>
+                <Button size="xs" variant="ghost" disabled={!article}>
+                  <EditIcon />
+                </Button>
+              </DialogTrigger>
+              <DialogContent>
+                <DialogTitle>Edit Article Link</DialogTitle>
+                <div className="mt-2">
+                  <EditWikiNodeForm node={node} articleKey={articleKey} />
+                </div>
+              </DialogContent>
+            </Dialog>
+          )}
           <Button size="xs" variant="ghost" onClick={deleteWikiNode}>
             <Trash2Icon className="text-destructive" />
           </Button>
         </div>
       </div>
       <div className="mt-1">
         {isLoading || !article || !articleKey ? (
-          <SimpleLoader />
+          <p>Are you sure article exist</p>
```

Comment:
Chaud de dire plutôt : (et mettre le texte en gris)

```suggestion
          <p className="text-muted-foreground">Article not found. Are you sure it still exists?</p>
```